src/access_control.ruleset

```diff
@@ -106,99 +106,100 @@ ruleset access_control_ruleset
         }
     }
 
+    // Check if a non-stranger swiped their badge.
+    //
+    // Reacts to:
+    //      scan.badge_scan
+    // Changes:
+    //      person.badged
+    //
     {
-        // Check if non-stranger switped badge
-        //
-        // Reacts to:
-        //      scan.badge_scan
-        // Changes:
-        //      person.badged
-        //
         if (@badge_scan && !stranger)
         {
             badged = true;
         }
     }
 
+    // Check if someone is joining a Wifi network.
+    //
+    // Reacts to:
+    //      scan.joining_wifi
+    // Changes:
+    //      person.on_wifi
+    //
     {
-        // Check if someone is joining wifi network
-        //
-        // Reacts to:
-        //      scan.joining_wifi
-        // Changes:
-        //      person.on_wifi
-        //
         if (@joining_wifi)
         {
             person.on_wifi = true;
         }
     }
 
+    // Check if someone is leaving a Wifi network.
+    //
+    // Reacts to:
+    //      scan.leaving_wifi
+    // Changes:
+    //      person.on_wifi
+    //
     {
-        // Check if someone is leaving wifi network
-        //
-        // Reacts to:
-        //      scan.leaving_wifi
-        // Changes:
-        //      person.on_wifi
-        //
         if (@leaving_wifi)
         {
             person.on_wifi = false;
         }
     }
 
+    // Check if a vehicle entered a parking lot.
+    //
+    // Reacts to:
+    //      scan.vehicle_entering
+    // Changes:
+    //      person.parked
+    //
     {
-        // Check if vehicle is parking
-        //
-        // Reacts to:
-        //      scan.vehicle_entering
-        // Changes:
-        //      person.parked
-        //
         if (@vehicle_entering)
         {
             person.parked = true;
         }
     }
 
+    // Check if a vehicle is departing from a parking lot.
+    //
+    // Reacts to:
+    //      scan.vehicle_departing
+    // Changes:
+    //      person.parked
+    //
     {
-        // Check if vehicle is departing
-        //
-        // Reacts to:
-        //      scan.vehicle_departing
-        // Changes:
-        //      person.parked
-        //
         if (@scan.vehicle_departing)
         {
 		    person.parked = false;
         }
     }
 
+    // Somebody scanned their face but lacks credentials.
```

Comment:
```suggestion
    // Handles when an inadmissible person scans their face.
```
Wondering if instead of inadmissible we might want to say:
lacking proper credentials
or
lacks sufficient access

Reply:
"Lacking proper credentials" sounds good.
```suggestion
    // Handles when a person scans their face but lacks proper credentials.
```